.ci/scripts/test_int_util.sh

```diff
@@ -68,53 +80,59 @@ test_int_setup_data_tmp_directory()
 {
     # Create a temporary directory to allow Jenkins to write to it and avoid collisions
     # with other users
-    TMP_DIR=$(mktemp -dp /data/tmp)
+    TMP_DIR=$(mktemp -dp ${TMP_ROOT})
     chmod 775 $TMP_DIR
-    cd $TMP_DIR
 }
 
 test_int_setup_test_data()
 {
     # Download test data and extract it
     #
-    local_testdata_archive=${TMP_DIR}/${TESTDATA}
+    local_input_data_archive=${TMP_DIR}/${INPUT_DATA}
+    local_expected_data_archive=${TMP_DIR}/${EXPECTED_DATA}
     local_runconfig=${TMP_DIR}/${RUNCONFIG}
 
     # Pull in test data and runconfig from S3
-    echo "Downloading test data from s3://operasds-dev-pge/${PGE_NAME}/${TESTDATA} to $local_testdata_archive"
-    aws s3 cp s3://operasds-dev-pge/${PGE_NAME}/${TESTDATA} $local_testdata_archive --no-progress
-    echo "Downloading runconfig from s3://operasds-dev-pge/${PGE_NAME}/${RUNCONFIG} to $local_runconfig"
-    aws s3 cp s3://operasds-dev-pge/${PGE_NAME}/${RUNCONFIG} $local_runconfig
-
-    cd ${TMP_DIR}
+    echo "Downloading input data from s3://operasds-dev-pge/${PGE_NAME}/${INPUT_DATA} to $local_input_data_archive"
+    aws s3 cp s3://operasds-dev-pge/${PGE_NAME}/${INPUT_DATA} $local_input_data_archive --no-progress
```

Comment:
When running locally, is there some setup required to get this to work?  If so maybe at a comment that points to the wiki (and of course a quick write up on the wiki)